js/app.js

```diff
@@ -26,3 +26,10 @@ if(window.mc) {
   };
 }
 
+// Setup FAQ item toggling
+var toggles = document.querySelectorAll('.toggle');
```

Comment:
FWIW we have jQuery on here, this whole 6 lines of javascript could be just

```
$('.toggle').on('click', function() { $(this).toggleClass('closed'); });
```

This works too though, no big